x-pack/plugins/stack_connectors/public/connector_types/stack/torq/torq_params.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  ActionParamsProps,
+  JsonEditorWithMessageVariables,
+} from '@kbn/triggers-actions-ui-plugin/public';
+import React from 'react';
+
+import { TorqActionParams } from '../../types';
+import * as i18n from './translations';
+
+const TorqParamsFields: React.FunctionComponent<ActionParamsProps<TorqActionParams>> = ({
+  actionParams,
+  editAction,
+  index,
+  messageVariables,
+  errors,
+}) => {
+  const { body } = actionParams;
+  return (
+    <JsonEditorWithMessageVariables
+      messageVariables={messageVariables}
+      paramsProperty={'body'}
+      inputTargetValue={body}
+      label={i18n.BODY_FIELD_LABEL}
+      aria-label={i18n.BODY_FIELD_ARIA_LABEL}
+      errors={errors.body as string[]}
+      onDocumentsChange={(json: string) => {
+        editAction('body', json, index);
+      }}
+      onBlur={() => {
+        if (!body) {
+          editAction('body', '', index);
+        }
+      }}
+    />
```

Comment:
I have found a way to not show the error decorations in the JsonEditor, we can overload the default props this way, it is a little bit hacky, but it will work. we show/hide the errors in the editor in accordance with the errors we get from our own validation.
The rest of the options are the same values as defined in the editor by default.


```suggestion
      euiCodeEditorProps={{
        options: {
          renderValidationDecorations: body && errors?.body?.length ? 'on' : 'off',
          lineNumbers: 'on',
          fontSize: 14,
          minimap: {
            enabled: false,
          },
          scrollBeyondLastLine: false,
          folding: true,
          wordWrap: 'on',
          wrappingIndent: 'indent',
          automaticLayout: true,
        },
      }}
    />
```

Reply:
Applied your solution, thank you!